build.sbt

```diff
@@ -40,7 +40,8 @@ lazy val core = (project in file("core"))
   .settings(
     name := repo,
     description := "Simple, safe and intuitive I/O in Scala",
-    libraryDependencies += Dependencies.scalaReflect(scalaVersion.value) % Provided
+    libraryDependencies += Dependencies.scalaReflect(scalaVersion.value) % "provided,optional",
+    libraryDependencies ++= Dependencies.silencer,
```

Comment:
why do we need silencer?

Reply:
* scalac generates bogus warnings of unused pattern variables when matching against a quasiquote.
* It generates (correct) warnings of unused parameters in the stub methods `at` & `from`. The parameters are only ever used by the macros, not by the methods themselves.